pages/common/vagrant-halt.md

```diff
@@ -0,0 +1,21 @@
+# vagrant halt
+
+> Shuts down the running machine Vagrant is managing.
+> See also: `vagrant`, `vagrant box`, `vagrant plugin`, `vagrant validate`.
+> More information: <https://developer.hashicorp.com/vagrant/docs/cli/halt>.
+
+- Halt the currently running Vagrant machine gracefully:
+
+`vagrant halt`
+
+- Halt a specific machine by its ID or name gracefully:
+
+`vagrant halt {{[id|name]}}`
+
+- Forcefully halt the current running machine(s) (This can affect multiple running machines if they are part of the same Vagrant environment):
+
+`vagrant halt {{[-f|--force]}}`
+
+- Forcefully halt a specific machine by its ID or name:
+
+`vagrant halt {{[-f|--force]}} {{[id|name]}}`
```

Comment:
```suggestion
`vagrant halt {{id_or_name}}`

- Forcefully halt the current running machine(s) (This can affect multiple running machines if they are part of the same Vagrant environment):

`vagrant halt {{[-f|--force]}}`

- Forcefully halt a specific machine by its ID or name:

`vagrant halt {{[-f|--force]}} {{id_or_name}}`
```

Not sure if this is the envisioned use-case for the new {{[]}} syntax. I was under the impression that it would be used for short/longform options.